plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeDelete.java

```diff
@@ -67,9 +67,12 @@ public void testTargetedDeleteWhenTableIsPartitionedWithColumnContainingSpecialC
                 "AS VALUES " +
                 "(1, 'with-hyphen'), " +
                 "(2, 'with:colon'), " +
-                "(3, 'with?question')", 3);
+                "(3, 'with:colon'), " +
```

Comment:
why `with:colon` twice?

Reply:
By having `with:colon` twice we get a parquet file with two rows. When we next delete one of these rows we trigger a "rewrite" of the parquet file, and corresponding remove and add entries in the delta log. If we only have a single row in the parquet file we would not need to do a rewrite of the parquet file and just add a remove entry in the delta log. Previously the code path which handles rewrites of parquet files did not have test coverage for special characters.

Reply:
Capture please the gist of this a code comment

Reply:
Added a comment and an assertion that the rows are in a single parquet file.